src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.cs

```diff
@@ -75,13 +80,17 @@ private static bool IsEccAlgorithmGroup(CngAlgorithmGroup algorithmGroup)
         private CngAlgorithmCore _core;
 
         // See https://msdn.microsoft.com/en-us/library/windows/desktop/bb931354(v=vs.85).aspx
+        // See https://msdn.microsoft.com/en-us/library/windows/desktop/mt632245%28v=vs.85%29.aspx for named curves
         private static readonly KeySizes[] s_legalKeySizes =
             new KeySizes[]
             {
                 // All values are in bits.
-                new KeySizes(minSize: 256, maxSize: 384, skipSize: 128),
-                new KeySizes(minSize: 521, maxSize: 521, skipSize: 0),
+                new KeySizes(minSize: 160, maxSize: 224, skipSize: 32), // 160, 192, 224
```

Comment:
This is going to be a real problem; I think.  We probably want to freeze set_KeySize changing the curve to its current behavior of 3 supported curves (and perhaps knock out the 224 support in the OpenSsl version), and make LegalKeySizes just be something like (160, int.MaxValue, 1).

So we can set base.KeySize to be whatever the KeySize is, and then we just use the curve definition internally.